.gitignore

```diff
@@ -1,5 +1,6 @@
 .gradle
 build/
+out/
```

Comment:
Huh I also got those in other projects, who's generating that folder?

Reply:
Intellij, running tests from the IDE.